src/components/presentational/DumbbellChart/DumbbellChart.tsx

```
@@ -2,74 +2,73 @@ import "./DumbbellChart.css";
 
-export interface ClosedInterval { values : number[]};
+export interface ClosedInterval { values: number[] };
 
 export interface DataPoint {
-    dataSet1 : ClosedInterval;
-    dataSet2 : ClosedInterval;
+    dataSet1: ClosedInterval;
+    dataSet2: ClosedInterval;
 }
 
 export interface Dumbbell {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Dumbbell'. Consider to delete it or rename it.

'Dumbbell' is defined here:


(lint/suspicious/noRedeclare)